test.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash -xe
 
-go test ./...
+go test -timeout 100ms ./...
```

Comment:
This timeout is too low

We have tests that have `Eventually` or `Consistently` that take longer to run.. let's change this to 1 second

Reply:
I upped it to 2s and forgot to push. I agree that 100ms is way too low.